src/Microsoft.ML/Runtime/EntryPoints/TrainTestMacro.cs

```diff
@@ -23,8 +24,14 @@ public sealed class SubGraphInput
 
         public sealed class SubGraphOutput
         {
-            [Argument(ArgumentType.Required, HelpText = "The model", SortOrder = 1)]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "The model", SortOrder = 1)]
             public Var<IPredictorModel> Model;
+            
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Transform model", SortOrder = 2)]
+            public Var<ITransformModel> TransformModel;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Indicates to use transform model instead of predictor model.", SortOrder = 3)]
+            public bool UseTransformModel = false;
```

Comment:
Unrelated to your code, but can anyone explain what's the difference between IPredictorModel and ITransformModel? or maybe related, because HelpText not helping at all. "The model" vs "Transform Model". What should I choose? hmmm....

Reply:
I'll change The Model to The predictor model.

---
In reply to: [191004407](https://github.com/dotnet/machinelearning/pull/212#discussion_r191004407) [](ancestors = 191004407)